lib/iris/tests/unit/analysis/trajectory/test__nearest_neighbour_indices_ndcoords.py

```diff
@@ -7,18 +7,15 @@
 
 """
 
-# Import iris.tests first so that some things can be initialised before
-# importing anything else.
-import iris.tests as tests  # isort:skip
-
 import numpy as np
+import pytest
 
 from iris.analysis.trajectory import _nearest_neighbour_indices_ndcoords as nn_ndinds
 from iris.coords import AuxCoord, DimCoord
 from iris.cube import Cube
 
 
-class Test2d(tests.IrisTest):
+class Test2d:
     def test_nonlatlon_simple_2d(self):
         co_y = DimCoord([10.0, 20.0], long_name="y")
```

Comment:
DO NOT ACTION, OUT OF SCOPE.

The first five lines of each test in this class (excluding comments) is duplicated, I think it should be moved into a _setup fixture.

Reply:
https://github.com/SciTools/iris/issues/6625